apps/docs-snippets/src/guide/predicates/predicates-with-configurable.test.ts

```diff
@@ -0,0 +1,75 @@
+import { WalletUnlocked, Predicate, NativeAssetId, BN, getRandomB256 } from 'fuels';
+
+import { SnippetContractEnum, getSnippetContractArtifacts } from '../../../contracts';
+import { getTestWallet } from '../../utils';
+
+describe(__filename, () => {
+  let wallet: WalletUnlocked;
+
+  const { abi, bin } = getSnippetContractArtifacts(
+    SnippetContractEnum.WHITELISTED_ADDRESS_PREDICATE
+  );
+
+  beforeAll(async () => {
+    wallet = await getTestWallet();
+  });
+
+  it('should successfully tranfer to setted whitelisted address', async () => {
+    // #region predicates-with-configurable-constants-2
+    const newWhitelistedAddress = getRandomB256();
+
+    const configurable = { WHITELISTED: newWhitelistedAddress };
+
+    // instantiate predicate with configurable constants
+    const predicate = new Predicate(bin, abi, wallet.provider, configurable);
+
+    // set predicate data to be the same as the configurable constant
+    predicate.setData(configurable.WHITELISTED);
+
+    // transfering funds to the predicate
+    const tx1 = await wallet.transfer(predicate.address, 500);
+
+    await tx1.waitForResult();
+
+    const destination = WalletUnlocked.generate();
```

Comment:
I think `wallet` would fit better than `destination`. If you'd like to be very explicit, you could even use `destinationWallet`, but since only one wallet is in context, `wallet` sounds enough.